### DISCUSSION TITLE
Allow for MM-DD or DD-MM format

### DISCUSSION THREAD

**Discussion by sweharris** [InfiniTimeOrg/InfiniTime#1590] [Ideas]
This appeared in the new Casio watchface; it has coded if in 12 hour mode then display MM-DD else display DD-MM.    So it's impossible to have a 24hr time but in MM-DD mode  (my preferred format).

```
      if (settingsController.GetClockType() == Controllers::Settings::ClockType::H24) {
...
        lv_label_set_text_fmt(label_date, "%3d-%2d", day, month);
...
      } else {
        lv_label_set_text_fmt(label_date, "%3d-%2d", month, day);
...
      }
```

Ideally we need a new Controller Setting to allow for the date order to be defined, and watch faces will read that.  Today this may be the only watchface with this issue, but as more  faces are built we might see more.

**Comment by ITCactus** [comment]
@sweharris  actually that feature has been requested as #1166.
but it's great, that more users want this :)

**Comment by ryspace** [comment]
I had thought this was going to be a part of the 1.12 update. I didn't know it was still being debated/discussed. Anyway, consider this a plus one here. I'd love to be able to have the US format on the Casio G7710 face with 24-hour mode.

**Comment by jonathonmckay** [comment]
I would also like to see a system wide setting for the date format. Each watch face currently uses a different format. I prefer dates in ISO 8601 (YYYY-MM-DD) format, but only the terminal watch face shows that (which is a bit too small for me to read).
